cases-SEP24/transactions.test.js

```diff
@@ -145,6 +145,39 @@ describe("Transactions", () => {
     expect(json.transactions.length, logs).toBe(1);
   });
 
+  it("returns a transactions in decending order", async () => {
+    await createTransaction({
+      currency: enabledCurrency,
+      account: keyPair.publicKey(),
+      toml: toml,
+      jwt: jwt,
+      isDeposit: true,
+    });
+    await createTransaction({
+      currency: enabledCurrency,
+      account: keyPair.publicKey(),
+      toml: toml,
+      jwt: jwt,
+      isDeposit: true,
+    });
+
+    const { json, status, logs } = await loggableFetch(
+      transferServer + `/transactions?asset_code=${enabledCurrency}`,
+      {
+        headers: {
+          Authorization: `Bearer ${jwt}`,
+        },
+      },
+    );
+    let expected = json.transactions.sort((a, b) =>
+      a.started_at > b.started_at ? 1 : -1,
+    );
+    expect(status, logs).toEqual(200);
+    expect(json.error, logs).not.toBeDefined();
+    expect(json.transactions.length, logs).toBeGreaterThanOrEqual(2);
+    expect(json.transactions, logs).toStrictEqual(expected);
```

Comment:
This is going to compare the entire structure which I guess is fine but I think a better way would be to expect the `id`'s in each list to ensure they match.

Reply:
Just made the change here.
 https://github.com/stellar/transfer-server-validator/pull/169/commits/f161cb006bdb061ae83950649e15459a11408598